src/test/java/wooteco/subway/auth/AuthServiceTest.java

```diff
@@ -0,0 +1,48 @@
+package wooteco.subway.auth;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import wooteco.subway.auth.application.AuthService;
+import wooteco.subway.auth.dto.TokenRequest;
+import wooteco.subway.member.application.MemberService;
+import wooteco.subway.member.dao.MemberDao;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@SpringBootTest
+public class AuthServiceTest {
+
+    @Autowired
+    private AuthService authService;
+
+    @MockBean
+    private MemberDao mockMemberDao;
+
+    private final String testEmail = "test";
+    private final String testPassword = "test";
+
+    @DisplayName("db에 존재하는 사용자로 token을 생성할 경우 예외를 발생한다.")
+    @Test
+    public void checkIsExisting(){
+        final TokenRequest tokenRequest = new TokenRequest(testEmail, testPassword);
+
+        Mockito.when(mockMemberDao.isExist(testEmail, testPassword))
+                .thenReturn(true);
+
+        authService.createToken(tokenRequest);
+    }
+
+    @DisplayName("db에 존재하지 않는 사용자로 token을 생성할 경우 예외를 발생한다.")
+    @Test
+    public void checkIsExistingWhenDataNotExisting(){
+        final TokenRequest tokenRequest = new TokenRequest(testEmail, testPassword);
+
+        assertThatThrownBy(()->{
+            authService.createToken(tokenRequest);
+        }).isInstanceOf(IllegalArgumentException.class);
+    }
```

Comment:
```suggestion
    @Test
    public void checkIsExistingWhenDataNotExisting() {
        final TokenRequest tokenRequest = new TokenRequest(testEmail, testPassword);

        assertThatThrownBy(() -> {
            authService.createToken(tokenRequest);
        }).isInstanceOf(IllegalArgumentException.class);
    }
```

1. 포맷팅 신경 써 주세요.
2. 테스트.. 돌려 보셨나요? 😭

Reply:
1. 수정 했습니다 😰
2. 지하철 경로 조회를 제외한 모든 테스트 케이스가 통과하도록 했습니다. 각 테스트 클래스에 `@DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)`를 붙여줘서 한 번에 테스트코드가 정상적으로 작동하도록 했습니다. 원래 코드에는 `AcceptanceTest`클래스에 `@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)`가 있었는데 이렇게 하니까 전체 테스트 코드 동작시 오류가 발생했습니다. 이유를 아직 못찾았습니다.. 테스트시 애플리케이션 컨텍스트 사용에 대해서 더 공부해 보려고 합니다!!